koku/api/provider/view.py

```diff
@@ -124,6 +125,9 @@ def create(self, request, *args, **kwargs):
                 }
             }
         """
+        if request.user.is_superuser:
```

Comment:
Should this be `if not` ?

Reply:
Provider creation should be restricted to non-service admin users.  The permissions set on this class is only IsAuthenticated.  So what we really need is a permissions set that is IsAuthenticated and not IsAdmin.

I'm not thrilled with this change but wanted to get the problem fixed with a test in place until I can get a better handle on how to make custom permissions in Django rest_framework.

I'm fine with holding off on this PR until we get a more robust solution in place.  What do you think?